examples/arm/setup.sh

```diff
@@ -65,8 +65,7 @@ tosa_reference_model_rev="70ed0b40fa831387e36abdb4f7fb9670a3464f5a"
 
 # vela
 vela_repo_url="https://gitlab.arm.com/artificial-intelligence/ethos-u/ethos-u-vela"
-vela_rev="e131bf4f528f0d461868229972e07f371dcbc881"
-
+vela_rev="46d88f56902be0706e051c10153ffb7620e01ee3"
```

Comment:
Let's add why we are updating this in the summary. 🙏 
With this I need to pull this internally as well.

Reply:
Do you need more then this?
-----
vela version is stepped to fix numerical issues with REDUCE_SUM.

Reply:
Sorry I didn't see this. No that is sufficient.